packages/web3-eth-accounts/src/account.ts

```diff
@@ -154,9 +155,13 @@ export const parseAndValidatePrivateKey = (data: Bytes, ignoreLength?: boolean):
  * web3.eth.accounts.hashMessage(web3.utils.utf8ToHex("Hello world")) // Will be hex decoded in hashMessage
  *
  * > "0x8144a6fa26be252b86456491fbcd43c1de7e022241845ffea1c3df066f7cfede"
+ *
+ * web3.eth.accounts.hashMessage("Hello world", true)
+ *
+ * > "0xed6c11b0b5b808960df26f5bfc471d04c1995b0ffd2055925ad1be28d6baadfd"
  * ```
  */
-export const hashMessage = (message: string): string => {
+export const hashMessage = (message: string, skipPrefix?: boolean): string => {
```

Comment:
above will work but using a default parameter is generally preferable in this case
```suggestion
export const hashMessage = (message: string, skipPrefix: boolean = false): string => {
```